modules/peridynamics/src/kernels/MechanicsBasePD.C

```diff
@@ -58,9 +57,9 @@ MechanicsBasePD::prepare()
   for (unsigned int nd = 0; nd < _nnodes; ++nd)
     _ivardofs[nd] = _current_elem->node_ptr(nd)->dof_number(_sys.number(), _var.number(), 0);
 
-  for (unsigned int i = 0; i < _dim; ++i)
-    _current_vec(i) = _origin_vec(i) + _disp_var[i]->getNodalValue(*_current_elem->node_ptr(1)) -
-                      _disp_var[i]->getNodalValue(*_current_elem->node_ptr(0));
+  for (_i = 0; _i < _dim; ++_i)
```

Comment:
What's the benefit of using the member variables of the base class for indexing here instead of local variables?

Reply:
I was thinking of avoiding using new index variable. It seems to me now that using the member variable from the base class sometimes will be problematic. Will discard those changes.